sdk/search/azure-search/setup.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
Are the files under `sdk/search/azure-search` for the namespace package?

Reply:
Yes, there is (probably) going to be a separate package for a separate service client in later previews (this is just the "query" client). Though ultimately it depends somewhat on whether the swaggers get merged and somewhat on our own intentions.

@johanste do you have thoughts about this question?

Reply:
Actually not sure we need this package after all. Removing for now.